doc/whats_new/v0.23.rst

```diff
@@ -51,6 +51,14 @@ Changelog
   more memory efficient implementation of single linkage clustering.
   :pr:`11514` by :user:`Leland McInnes <lmcinnes>`.
 
+:mod:`sklearn.linear_model`
+...........................
+
+- |Fix| Fixed a bug that made :class:`linear_model.RANSACRegressor` fail when
```

Comment:
```suggestion
- |Fix| Fixed a bug that made :class:`linear_model.RANSACRegressor` failed when
```

Reply:
If we remove all the extra info around RANSACRegressor, the original reads:
Fixed a bug that made RANSACRegressor fail when

The suggestion reads:
Fixed a bug that made RANSACRegressor failed when

The original grammar is correct, are you saying this is just the standard? Should I change something else?

Reply:
Sorry my mistake

Reply:
The bug is not around failing if the estimator requires weights. It's that weights should have been passed on any case.

"Fixed a bug where sample_weight were not used when fitting the final estimator ..."

Reply:
Ah, right. I corrected it.